apps/src/turtle/turtle.js

```diff
@@ -482,7 +482,9 @@ Artist.prototype.drawTurtle = function() {
       sourceY + sourceHeight > this.avatarImage.height)
   {
     if (console && console.log) {
-      console.log("drawImage is out of source bounds!");
+      // TODO(bjordan): ask Brent, starting to flood grunt mochaTest messages,
+      // better fix here?
+      // console.log("drawImage is out of source bounds!");
```

Comment:
I'm not positive, but one theory is that the change I made here to runLevelTest's `baseUrl` to support asset serving ([here](https://github.com/code-dot-org/code-dot-org/blob/2015-mc-hoc/apps/test/util/runLevelTest.js#L122-122)) might have caused the avatar image to actually get served?

baseUrl in that file used to be set to `baseUrl: '' // doesn't matter`